components/admin_console/admin_definition.jsx

```diff
@@ -5224,7 +5328,19 @@ const AdminDefinition = {
                         label: t('admin.service.overrideTitle'),
                         label_default: 'Enable integrations to override usernames:',
                         help_text: t('admin.service.overrideDescription'),
-                        help_text_default: 'When true, webhooks, slash commands and other integrations, such as [Zapier](!https://developers.mattermost.com/integrate/admin-guide/admin-zapier-integration/), will be allowed to change the username they are posting as. Note: Combined with allowing integrations to override profile picture icons, users may be able to perform phishing attacks by attempting to impersonate other users.',
+                        help_text_default: 'When true, webhooks, slash commands and other integrations, such as {Zapier}, will be allowed to change the username they are posting as. Note: Combined with allowing integrations to override profile picture icons, users may be able to perform phishing attacks by attempting to impersonate other users.',
+                        help_text_values: {
+                            documentation: (
+                                <a href='https://developers.mattermost.com/integrate/admin-guide/admin-zapier-integration/'>
```

Comment:
The placeholder link has been removed from the associated string above. This code may no longer be necessary.